### PR TITLE
[stable/sonarqube] Add ability to add/mount extra volumes

### PR DESCRIPTION
**Pull request by sparqueur**
#### What this PR does / why we need it:
Add the ability to add/mount extra volumes

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by sparqueur** [comment]
@tsiddique  @rjkernick  Do you have an idea when you might be able to review this PullRequest ?
Thanks in advance,

File: stable/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 3.2.6
+version: 3.2.7
```
Comment (rjkernick) [inline stable/sonarqube/Chart.yaml]: Please bump the minor version because this adds features.

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Comment by sparqueur** [comment]
@rjkernick Done => 3.2.8

**Comment by sparqueur** [comment]
All should be good, sorry I had issue with gpd signature

**Comment by sparqueur** [comment]
Hi, Is there any progress about this PR ?

**Pull request merged**